### PR TITLE
Desc/iconifies various pills, admin floorpill bottles

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
tweak: Contraband pills have been given a unified pill icon as well as new names and descriptions. An admin spawn only bottle of floorpills has been added.
/:cl:

- The "roundstart" var on pills has been changed to "rename_with_volume" to better describe what it actually does.

I was working on smuggler's satchels and thought that the contraband pills could do with a little bit of a stronger theme.

### CODE REVIEW

**Comment by Vidiveni** [comment]
Please don't rename old established chems if not necessary. I think the new name sounds bad and its motivation farfetched. A smuggler would probably care more about stealth than expect disabler spam. 

How about a bar drink that lets you silently use some tools, like to lift floor tiles?

**Comment by adamkad1** [comment]
aranesp is such a lazy name tho

**Comment by anconfuzedrock** [comment]
Smuggler's run is a dumb name tho

**Comment by Tlaltecuhtli** [comment]
how do you spell aranesp

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -83,135 +83,152 @@
...
-	roundstart = TRUE
+	rename_with_volume = TRUE
+
 /obj/item/reagent_containers/pill/cyanide
 	name = "cyanide pill"
 	desc = "Don't swallow this."
 	icon_state = "pill5"
 	list_reagents = list("cyanide" = 50)
-	roundstart = TRUE
+
```
Comment (Jordie0608) [inline code/modules/reagents/reagent_containers/pill.dm]: Is this supposed to be removed?

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -83,135 +83,152 @@
...
 /obj/item/reagent_containers/pill/adminordrazine
 	name = "adminordrazine pill"
 	desc = "It's magic. We don't have to explain it."
 	icon_state = "pill16"
 	list_reagents = list("adminordrazine" = 50)
-	roundstart = TRUE
+
```
Comment (Jordie0608) [inline code/modules/reagents/reagent_containers/pill.dm]: Same.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -83,135 +83,152 @@
...
-	roundstart = TRUE
+	rename_with_volume = TRUE
+
 /obj/item/reagent_containers/pill/stimulant
 	name = "stimulant pill"
 	desc = "Often taken by overworked employees, athletes, and the inebriated. You'll snap to attention immediately!"
 	icon_state = "pill19"
 	list_reagents = list("ephedrine" = 10, "antihol" = 10, "coffee" = 30)
-	roundstart = TRUE
+
```
Comment (Jordie0608) [inline code/modules/reagents/reagent_containers/pill.dm]: Same.

**Review by Jordie0608 — COMMENTED** [review]
Probably keep the name Aranesp.

**Comment by ExcessiveUseOfCobblestone** [comment]
who is writing this lore?!!!!

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Revert the rename, and whats with all the `roundstart`/`rename_with_volume` changes?

**Comment by coiax** [comment]
The "roundstart/rename_with_volume" var has the pill's name changed to its effective volume, and I've taken it away from pills that wouldn't be produced and labeled in the standard medical way, like adminrozine, or the stimulant pills taken by atheletes.

**Comment by ExcessiveUseOfCobblestone** [comment]
granny pr

**Comment by coiax** [comment]
Do people not like smuggler's run as the name for the reagent, or for both the reagent and the name for the contraband pill? Can someone suggest a better name for the contraband pill then?

**Comment by ExcessiveUseOfCobblestone** [comment]
what's wrong with it

**Comment by coiax** [comment]
I don't understand, Cobby, what would you like to change?

**Comment by ExcessiveUseOfCobblestone** [comment]
i'm asking what's wrong with the current name that warrants a change within this pr

**Comment by coiax** [comment]
Okay, I'll change it back.

**Pull request merged**